keel-core/src/main/kotlin/com/netflix/spinnaker/keel/serialization/ObjectMappers.kt

```diff
@@ -32,6 +33,7 @@ private fun <T : ObjectMapper> T.configureMe(): T =
       .registerULIDModule()
       .registerModule(JavaTimeModule())
       .configureSaneDateTimeRepresentation()
+      .disable(FAIL_ON_UNKNOWN_PROPERTIES)
```

Comment:
I think it doesn't hurt to have this set. It means I don't need to create a migration to remove the `current` and `desired` properties from persisted instances of `ResourceDeltaResolved`.